akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/Behaviors.scala

```diff
@@ -74,11 +74,11 @@ object Behaviors {
    * shall terminate voluntarily. If this actor has created child actors then
    * these will be stopped as part of the shutdown procedure.
    *
-   * The PostStop signal that results from stopping this actor will be passed to the
-   * given `postStop` behavior. All other messages and signals will effectively be
-   * ignored.
+   * The `PostStop` signal that results from stopping this actor will first be passed to the
+   * current behavior and then the provided `postStop` callback will be invoked.
+   * All other messages and signals will effectively be ignored.
    */
-  def stopped[T](postStop: Behavior[T]): Behavior[T] = Behavior.stopped(postStop)
+  def stopped[T](postStop: Effect): Behavior[T] = Behavior.stopped(postStop.apply _)
```

Comment:
This is an additional improvement, that is not 100% overlapping with the issue, but made the issue easier to solve - before the callback for  `stopped` was a behavior that would only ever get the `PostStop` signal, now it is a callback.

Reply:
Nice.